go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	bitbucket.org/bertimus9/systemstat v0.5.0
 	github.com/JeffAshton/win_pdh v0.0.0-20161109143554-76bb4ee9f0ab
 	github.com/Microsoft/go-winio v0.6.0
-	github.com/Microsoft/hcsshim v0.8.25
+	github.com/Microsoft/hcsshim v0.8.26
```

Comment:
After this update, do we now have a path forward to move to the latest versions of hcsshim (https://github.com/microsoft/hcsshim/releases/tag/v0.12.4)?

Reply:
> After this update, do we now have a path forward to move to the latest versions of hcsshim (https://github.com/microsoft/hcsshim/releases/tag/v0.12.4)?

Including latest version of hcsshim involves lot many changes. The diff is too high. We are also thinking of options to separate out hcn and hns codes from hcsshim as we are only using those packages in kubernetes from the big hccshim. The discussion is ongoing.